packages/invoices/src/types.ts

```diff
@@ -19,25 +19,6 @@ import {
  * TODO: Maybe we can get away with just using `Shipment` for both.
  */
 
-export type OutboundShipmentStatus =
```

Comment:
I've removed these and just using `InvoiceNodeStatus`.. ideally we could use separate types for inbound/outbound but it causes some annoyances. Kind of wish there were separate queries for inbound/outbound and different inbound/outbound types rather than a single `InvoiceNode` 🤷

Reply:
this is still being used in `utils` - is there a commit missing?

Reply:
Hmm now I'm not sure if I just missed this and was allowed to commit or if I did miss a commit. Fixed now regardless!